mod/bookmarks/views/default/bookmarks/bookmarklet.js

```diff
@@ -0,0 +1,3 @@
+location.href = BASEURL +
```

Comment:
the newlines don't cause problems in the output of https://github.com/Elgg/Elgg/pull/11507/files#diff-3e4217c9c8957a1571b6c9115a72e21cR31